dev/com.ibm.ws.springboot.fat20.mbean.app/bnd.bnd

```diff
@@ -0,0 +1,51 @@
+#*******************************************************************************
+# Copyright (c) 2017, 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= \
+  ~../cnf/resources/bnd/bundle.props, \
+  ~../cnf/resources/bnd/bundle.spring.props
+-nouses=true
+
+bVersion=1.0
+
+test.project: true
+instrument.disabled: true
+publish.wlp.jar.disabled: true
+
+src: \
+  src/main/java, \
+  src/main/resources
+
+# Specify bundles to be added to the classpath of a VM used in testing. 
+# Entries in the -testpath instruction enable the Eclipse Package Explorer 
+# to resolve compile dependencies for test classes. The instruction is not
+# required to build Spring Boot test artifacts. Each entry requies a 
+# corresponding entry in file cnf/oss_depenendcies.maven.
+# Ref: https://bndtools.org/manual/packageexplorer.html
+
+-testpath: \
+  org.springframework.boot:spring-boot;${springBootVersion20}, \
+  org.springframework.boot:spring-boot-autoconfigure;${springBootVersion20}, \
+\
+  org.springframework:spring-beans;${springVersion20}, \
+  org.springframework:spring-context;${springVersion20}, \
+  org.springframework:spring-core;${springVersion20}, \
+  org.springframework:spring-web;${springVersion20}, \
+\
+  com.ibm.websphere.javaee.servlet.3.1
+-buildpath: \
+	com.hazelcast,\
+	com.hazelcast,\
+	com.ibm.ws.org.slf4j.api,\
+	com.ibm.websphere.org.osgi.core,\
+	com.ibm.websphere.org.osgi.core,\
+	com.ibm.ws.org.slf4j.api
```

Comment:
I'm not sure you actually need the `-buildpath` and `-testpath` here.

Reply:
Thank you Tom! Yes, they've been removed